src/servers/main.cc

```diff
@@ -79,21 +82,26 @@ bool allow_model_control_ = false;
 // The HTTP, GRPC and metrics service/s and ports. Initialized to
 // default values and modifyied based on command-line args. Set to -1
 // to indicate the protocol is disabled.
+// Default to using the V1 protocol.
```

Comment:
You put this between the comment above that goes with the code below. Move this up above the comment.